tests/yast2_gui/yast2_control_center.pm

```diff
@@ -325,10 +325,10 @@ sub run {
         # see bsc#1062331, sound is not added to the yast2 pattern
         ensure_installed 'yast2-boot-server yast2-sound';
     }
-    elsif (is_tumbleweed) {
+    elsif (is_tumbleweed || is_leap('>15.3')) {
```

Comment:
```suggestion
    elsif (!is_sle || is_leap('>15.3')) {
```

Reply:
`!is_sle` implies `is_leap` is true, but I'm not sure we have the same issue on `Leap < 15.4`, we will see how it is on 15.3 QU3 Respin, if we had the same case on QU3, we can just leave it as `elif (!is_sle)`